SPECS/linux/0001-vmxnet3-version-4-driver.patch

```diff
@@ -0,0 +1,1047 @@
+From ba4f184e126b751d1bffad5897f263108befc780 Sun Sep 20 16:33:55 2020 -0700
+From: Naadir Jeewa <jeewan@vmware.com>
+Date: Thu, 24 Sep 2020 22:52:30 +0100
+Subject: Backport vmxnet3 from Linux 5.9-rc6
+
+Backport entire vmxnet3 from mainline Linux
```

Comment:
This is not the right method to backport patches to older kernels in Photon. We follow a model that's similar to how older stable kernels are maintained in Linux upstream. For details, you can check out: https://www.kernel.org/doc/html/v5.8/process/stable-kernel-rules.html

This means that you'll need to backport a specific commit or series of commits from mainline to 4.19 kernel, as individual patches, and each one of them should mention the corresponding upstream commit id as the first line in the commit message.

It would be much better if you can identify just the fixes and backport them, as opposed to backporting the entire driver. Further, any non-trivial changes you had to make to apply the backport to 4.19 should be explicitly called out in the patch's commit message.

You can use this change as a reference: https://github.com/vmware/photon/commit/f308bbb854e946503e193ed7e69247d3459b0e1a